packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.base.tsx

```diff
@@ -401,10 +401,10 @@ export class DropdownBase extends BaseComponent<IDropdownInternalProps, IDropdow
   }
 
   /** Render text in dropdown input */
-  private _onRenderTitle = (item: IDropdownOption[]): JSX.Element => {
+  private _onRenderTitle = (items: IDropdownOption[]): JSX.Element => {
```

Comment:
This technically isn't a match for the prop type and I'd hope TS would error on calling `map` on this arg.. I wonder if TS3.x would have caught this...

Reply:
It'll match with this change tho :)

Agreed that it is strange it wasn't caught by the compiler.